Catalyst.Fabric.Authorization.Client/Extensions/AuthorizationExtensions.cs

```diff
@@ -24,5 +26,15 @@ public static HttpRequestMessage AddContent<T>(this HttpRequestMessage httpReque
             httpRequestMessage.Content = new StringContent(JsonConvert.SerializeObject(model), Encoding.UTF8, mediaType);
             return httpRequestMessage;
         }
+
+        public static HttpClient FormatBaseUrl(this HttpClient client)
+        {
+            // so this will trim the forward slash whether its there or not.  it will 
+            // also fix any additional forward slashes.  Once it has trimmed the end,
+            // it will append the forward slash needed for making accurate calls.
+            // see also: https://stackoverflow.com/questions/23438416/why-is-httpclient-baseaddress-not-working#23438417
+            client.BaseAddress = new Uri($"{client.BaseAddress.OriginalString.TrimEnd('/')}/");

```

Comment:
we ended up doing the same thing in the Identity CLI

Reply:
its simple code to just make sure the right forward slash is there.  Kinda lame but i dont think it matters too much to make it simple.